<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Football Manager Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutCentral">
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTeams">
       <attribute name="title">
        <string>Команды</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTeams">
        <item>
         <widget class="QComboBox" name="teamComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="labelTeamDetails">
          <property name="text">
           <string>Детали команды будут отображены здесь</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetTeamPlayers"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMatches">
       <attribute name="title">
        <string>Матчи</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutMatches">
        <item>
         <widget class="QLabel" name="labelSchedule">
          <property name="text">
           <string>Следующий матч:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTransfers">
       <attribute name="title">
        <string>Трансферы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutTransfers">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutTransferTeams">
          <item>
           <widget class="QComboBox" name="comboBoxSellTeam">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxBuyTeam">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetTransferPlayers"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnTransferPlayer">
          <property name="text">
           <string>Перевести игрока</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSeason">
       <attribute name="title">
        <string>Сезон</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutSeason">
        <item>
         <widget class="QPushButton" name="btnNextSeason">
          <property name="text">
           <string>Завершить сезон</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
